adapters/networkHandlerFactory.js

```diff
@@ -14,12 +14,17 @@ const {
   networkHandler: GoogleAdwordsRemarketingListNetworkHandler
 } = require("../v0/destinations/google_adwords_remarketing_lists/util");
 
+const {
+  networkHandler: GoogleAdsEnhancedConversion
+} = require("../v0/destinations/google_ads_enhanced_conversion/util");
+
 const handler = {
   generic: GenericNetworkHandler,
   braze: BrazeNetworkHandler,
   marketo: MarketoNetworkHandler,
   pardot: PardotNetworkHandler,
-  google_adwords_remarketing_lists: GoogleAdwordsRemarketingListNetworkHandler
+  google_adwords_remarketing_lists: GoogleAdwordsRemarketingListNetworkHandler,
+  google_ads_enhanced_conversion: GoogleAdsEnhancedConversion
```

Comment:
What is the difference b/w the key (destination) names used here vs in features.json?

Reply:
Names are the same but in lower case here and uppercase in feature.json.

Reply:
We should have a single source of this data to avoid errors.
CC @sanpj2292 as this is tech debt.